src/modules/rlm_python/rlm_python.c

```diff
@@ -1106,9 +1139,37 @@ static int mod_load(void)
 	{
 		wchar_t *wide_name;
 		wide_name = Py_DecodeLocale(main_config->name, NULL);
+
```

Comment:
I'm skeptical python would merge these two configs correctly.  I think the easier way would be to move the python path into mod_load, and set it from a global configuration item, and not from the individual modules.  We didn't have support for global configuration items when a lot of this code was first developed.